samza-test/src/main/python/configs/tests.json

```diff
@@ -1,5 +1,5 @@
 {
-  "samza_executable": "samza-test_2.10-0.11.0-SNAPSHOT.tgz",
+  "samza_executable": "samza-test_2.10-0.10.1-SNAPSHOT.tgz",
```

Comment:
The latest samza release will go w/ samza-test_2.10-0.12.0-SNAPSHOT.tgz soon. Please check master and update accordingly.